src/cornerstone-core.js

```diff
@@ -0,0 +1 @@
+export * from '../../cornerstone/src/index.js';
```

Comment:
I don't think so .. I think it should still work as webpack still exports 'cornerstone-core' (which is different to 'src\cornerstone-core.js' which I added and is used internally. 

I added cornerstone-core.js so that if someone chooses to load modules directly with script type="module" but have cornerstone in a different path than cornerstoneTools, they only need to change the path in 'src\cornerstone-core.js' instead of in each file. This also made the changes I've done simpler and easier to understand/merge.
 
Since this is now available in both chrome beta and firefox beta it will soon be available in the official version and make developement much easier without the separate webpack stage.